### PR TITLE
Improve readme a bit (typos, punctuation, grammar, etc.)

### PR DESCRIPTION
**Pull request by mfn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,9 @@ Let say you have string with dates in format d.m.y or y-m-d separated by comma.
 
 ## Branch types
 
-You could declare previous grammar as PEG, it would improve speed x10. You can declare grammar as PEG by adding
```
Comment (mfn) [inline README.md:41]: This sentence was left unfinished

File: README.md [hunk 2]
```diff
@@ -62,19 +63,18 @@ Matches text. You can also use single quotes. You can use escape sequences so "\
 
 ##### /regular|expression/
 Matches given regular expression. You can use pattern modifiers.
-Grammar like "start :=> /[a-z]/i." will match also upper case letters.
-Regular expression cannot be backtracked. They work like fist match is only match
```
Comment (mfn) [inline README.md:66]: Typo ;-)

File: README.md [hunk 3]
```diff
@@ -83,23 +83,20 @@ When ignoreWhitespaces mode is on then /\s/, " ", "\t", "\n" won't work and you
 In ignoreWhitespaces mode these symbols check context and not consuming characters from input.
 For example sequence: 'a' newLine space space 'b' will match characters 'a' and 'b' separated by at least one space and at least one new line symbol
 
-##### text
```
Comment (mfn) [inline README.md]: I believe this is a duplication of `##### "text"` or I didn't understand the intention?

**Review by mfn — COMMENTED** [review]
While a I read the readme I found a typo and then some sentences where the grammar was a bit off.

I'm not a native English speaker either but maybe you like the changes.

File: README.md (hunk 2 above)
Comment (farafiri) [reply README.md:66]: Good eye :) Thank you.

**Pull request merged**